### ISSUE TITLE
FLuid voiding by Output Hatch (ME)

### ISSUE THREAD

**Issue by RaitGamerGR** [GTNewHorizons/GT-New-Horizons-Modpack#14556]
### Your GTNH Discord Username

Rait_GamerGR(Александр)#5721

### Your Pack Version

126# Nightly

### Your Server

Private Server

### Java Version

Java 8

### Type of Server

Ultramine

### Your Expectation

Trying to use Output Hatch (ME) as output hatch for Mega Chemical Reactor with batch mode

### The Reality

If multiblock will produce giant amounts of fluid, hatch will be looks like this, and fluid will be voided:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/70205464/95d1cfd7-ac32-442c-a66b-d7cf90188358)

### Your Proposal

Increase max amount of fluid 

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by vovaksenov** [comment]
I think rather than increasing the max fluid amount, me output hatches should obtain void prevention support like normal hatches already do. If the multi cant run a recipe without some fluid overflowing, it simply should not run. In best case scenario, the buffer size should be configurable to allow for smaller values at which the machine stops

**Comment by lordIcocain** [comment]
can you check if infinity cache enabled?

**Comment by Eldrinn-Elantey** [comment]
At the first use, I also thought that the liquid was destroyed, but I conducted one test. After installing hatch, I did not connect it to the network and started the recipe on the multiblock after several cycles, I connected hatch to the network and everything that should have arrived in it. So the infinite cache that is written about in tooltips works.

I had a case when it was written that there was not enough space for output despite the presence of subject and liquid outputs through the ME. When I allowed the removal of excess, the controller showed a different problem
